flight/Modules/System/systemmod.c

```diff
@@ -360,12 +447,18 @@ static void updateWDGstats()
 }
 #endif
 
+#ifdef PIPXTREME
```

Comment:
It's targeting the ground-side modem-- pipxtreme/oplink mini/taulinkmodule/etc which all run on the pipxtreme target.  The ground side modem should not play with alarms as the flight board is expected to do this.

You can buy compatible hardware here: http://www.aliexpress.com/item/Free-shipping-OPLINK-MINI-CC3D-REVO-Universal-Transceiver-TX-RX-Module-Integrating-Remote-Controller-PPM-Input/32431902268.html?spm=2114.01020208.3.1.orWplJ&ws_ab_test=searchweb201556_9_79_78_77_80,searchweb201644_5,searchweb201560_9

In general, I don't like the way we handle remote vs. local objects.  I don't like that it's not possible to use sparky2 as a ground-side modem, etc.  Factoring this in common helps the ultimate cleanup to make boards more versatile, but doesn't really take any direct steps towards it.